erasure-coding/src/lib.rs

```diff
@@ -258,7 +258,7 @@ where
 	// construct trie mapping each chunk's index to its hash.
 	{
 		let mut trie = TrieDBMut::new(&mut trie_storage, &mut root);
-		for (i, chunk) in chunks.as_ref().iter().enumerate() {
+		for (i, chunk) in chunks.iter().enumerate() {
```

Comment:
unrelated change

Reply:
I was getting this error:

```
error[E0282]: type annotations needed
   --> erasure-coding/src/lib.rs:261:28
    |
261 |         for (i, chunk) in chunks.as_ref().iter().enumerate() {
    |                           -------^^^^^^--
    |                           |      |
    |                           |      cannot infer type for type parameter `T` declared on the trait `AsRef`
    |                           this method call resolves to `&T`
    |
    = note: type must be known at this point
```

And on debugging saw the as_ref() made no difference to the object.